### DISCUSSION TITLE
Feature: Multiplayer map marking

### DISCUSSION THREAD

**Discussion by jimthehuman** [OpenRCT2/OpenRCT2#15683] [Feature requests]
It would help a lot in multiplayer if there was a way to place a temporary marker to help coordinate with building items. Me and my friends have a hard time trying to describe exactly where on the map to place certain items.

**Comment by Superjustinbros** [comment]
I support this.

**Comment by janisozaur** [comment]
This sounds like a feature to me.

One way to do that would be to utilise something like clickable link in chat, akin to https://github.com/OpenRCT2/OpenRCT2/pull/3168

Clicking on a message like `{X:1,Y:23}` would centre you on this tile, there would have to be some way of placing a ping with mouse cursor.

**Comment by Richard-L** [comment]
Generally the mouse cursors of other users should just be visible if in any way possible. The chat is great, but it's a pity you have no idea what others are doing.

**Comment by IntelOrca** [comment]
@Richard-L when you open a player window you can see a viewport focusing on their changes.

**Comment by wolfreak99** [comment]
One thing I've thought of lately to also help is, you know how when you place something that costs money, you see that wiggling cash value? And how if you hover over a ride you see the ride name? What if there was a text that would appear above someones action with their name? It would appear for 1 second (or maybe 750ms), if they do another action, the text for their name would lerp over to that position. It would filter out the users actions that are quicker than 300 Ms that way if they lay down a bunch of paths it won't just show a text shooting across the screen that would be too fast to read.
